### PR TITLE
LPS-133815 Create tests to user creator sort

### PR DESCRIPTION
**Pull request by MarinhoFeliphe**

### CODE REVIEW

**Comment by ramonmsp** [comment]
Just started reviewing :)

File: modules/dxp/apps/portal-workflow/portal-workflow-metrics-rest-test/src/testIntegration/java/com/liferay/portal/workflow/metrics/rest/resource/v1_0/test/InstanceResourceTest.java [hunk 1]
```diff
@@ -221,13 +225,58 @@ public void testGetProcessInstancesPageWithSortDateTime() throws Exception {
...
+					User user2 = _addUser("bbb");
+
+					instance2.setCreator(
+						new Creator() {
+							{
+								id = user2.getUserId();
+								name = user2.getFullName();
+							}
+						});
+				}
```
Comment (rafaprax) [inline modules/dxp/apps/portal-workflow/portal-workflow-metrics-rest-test/src/testIntegration/java/com/liferay/portal/workflow/metrics/rest/resource/v1_0/test/InstanceResourceTest.java:256]: Hey @MarinhoFeliphe, can we use the other overloaded `setCreator` option? The code should look like the code below:

```java
instance1.setCreator(
  () -> {
     User user = _addUser("aaa");
    
     return new Creator() {
	    {
		    id = user.getUserId();
		    name = user.getFullName();
	    }
      };
  });
```

Thx

File: modules/dxp/apps/portal-workflow/portal-workflow-metrics-rest-test/src/testIntegration/java/com/liferay/portal/workflow/metrics/rest/resource/v1_0/test/InstanceResourceTest.java [hunk 2]
```diff
@@ -345,6 +394,18 @@ protected Instance testPostProcessInstance_addInstance(Instance instance)
...
+			TestPropsValues.getCompanyId(), TestPropsValues.getUserId(),
+			RandomTestUtil.randomString(
+				NumericStringRandomizerBumper.INSTANCE,
+				UniqueStringRandomizerBumper.INSTANCE),
+			LocaleUtil.getDefault(), firstName,
+			StringUtil.toLowerCase(RandomTestUtil.randomString()),
+			new long[] {TestPropsValues.getGroupId()},
+			ServiceContextTestUtil.getServiceContext());
+	}
+
```
Comment (rafaprax) [inline modules/dxp/apps/portal-workflow/portal-workflow-metrics-rest-test/src/testIntegration/java/com/liferay/portal/workflow/metrics/rest/resource/v1_0/test/InstanceResourceTest.java:408]: Hey @MarinhoFeliphe, Can we use this overloaded `UserTestUtil.addUser` option? 

```java

public static User addUser(
	String screenName, Locale locale, String firstName, String lastName,
	long[] groupIds)
  throws Exception {
  
  return addUser(
	  TestPropsValues.getCompanyId(), TestPropsValues.getUserId(),
	  screenName, locale, firstName, lastName, groupIds,
	  ServiceContextTestUtil.getServiceContext());
}
````
Thx

**Review by rafaprax — CHANGES_REQUESTED** [review]

**Comment by MarinhoFeliphe** [comment]
ci:test:sf

**Comment by MarinhoFeliphe** [comment]
ci:test:sf

**Comment by MarinhoFeliphe** [comment]
ci:forward

**Pull request closed without merging**